web/packages/teleport/src/Users/Users.test.tsx

```diff
@@ -105,6 +107,59 @@ describe('invite collaborators integration', () => {
   });
 });
 
+describe('Users not equal to MAU Notice', () => {
```

Comment:
there is only one test inside this describe, i'd remove the describe block and move everything into the `test`?